package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "1.4.1",
+  "version": "1.5.0",
```

Comment:
since we're publishing an alpha, can we bump this to `1.5.0-alpha.423` instead?

Reply:
actually, we should revert this change and bump the package version in a separate pr since we have other changes in develop